server/index.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express')
+const app = express();
+const port = 8000;
+
+var project = [
+    {
+       "name":"All Notes"
+    }
+ ]
+ 
+
+app.get('/', (req, res) => {
+    res.json(project)
+  });
+  
+  app.listen(port, () => {
+    console.log(`Application listening on port ${port}!`)
+  });
+  
+  
```

Comment:
Can you add a new line at the file? 

I understand the point of what is a new line. It is just a line without any further spaces. This is because, `diff` programs (like used by git diff, Github) show line by line differences between files. They usually print a message when only one file ends with a newline because else you would not see this difference.

Check [this](https://thoughtbot.com/blog/no-newline-at-end-of-file) out for a bit of a history on this. It's super easy to change this though. 

Let me know on Slack if you need any help on this.